docs/spark-etl/spark-etl-run-examples.md

```diff
@@ -14,68 +14,204 @@ spark-submit \
 --master local[*] \
 --driver-memory 2G \
 $JAR \
---input hadoop --format {geotiff | temporal-geotiff} --cache NONE -I path="file:///Data/nlcd/tiles" \
---output s3 -O bucket=com.azavea.datahub key=catalog \
---layer nlcd-tms --crs EPSG:3857 --pyramid --layoutScheme {tms | floating}
+--credentials "file://credentials.json" \
+--datasets "file://datasets.json"
 ```
 
-### Ingest singleband tiles from local fs or hdfs into accumulo storage command
+### Credentials JSON example
+
+`credentials.json`:
+
+```json
+{
+  "accumulo": [{
+    "name": "accumulo-gis",
+    "zookeepers": "zookeepers",
+    "instance": "instance",
+    "user": "user",
+    "password": "password"
+  }],
+  "cassandra": [{
+    "name": "cassandra-gis",
+    "allowRemoteDCsForLocalConsistencyLevel": false,
+    "localDc": "datacenter1",
+    "usedHostsPerRemoteDc": 0,
+    "hosts": "hosts",
+    "replicationStrategy": "SimpleStrategy",
+    "replicationFactor": 1,
+    "user": "user",
+    "password": "password"
+  }],
+  "s3": [],
+  "hadoop": []
+}
+```
 
-```sh
-#!/bin/sh
-export JAR="geotrellis-etl-assembly-0.10-SNAPSHOT.jar"
 
-spark-submit \
---class geotrellis.spark.etl.{SinglebandIngest | MultibandIngest} \
---master local[*] \
---driver-memory 2G \
-$JAR \
---input hadoop --format {geotiff | temporal-geotiff} --cache NONE -I path="file:///Data/nlcd/tiles" \
---output accumulo -O instance="gis-accumulo-instance" table="nlcd-table" user="root" password="password" zookeeper="zoo1:2181,zoo2:2181,zoo3:2181" \
---layer nlcd-tms --crs EPSG:3857 --pyramid --layoutScheme {tms | floating}
+### Ingest tiles from local fs or hdfs into s3 storage command
+
+`datasets.json`:
```

Comment:
Would it make sense to have input dataset and output config to be in separate files? This way you can have multiple sources and only one destination like `--output azavea-datahub.json`